### DISCUSSION TITLE
How can I make this work?

### DISCUSSION THREAD

**Discussion by Gostar64** [KadeArchive/Kade-Engine#1829] [Q&A]
So in the mod chart I have this string of code
but when I try to run the mod chart, it comes with an error that says '<name>' expected near "boyfriend".
**help**

![image](https://user-images.githubusercontent.com/88996224/129813200-00f954ca-b163-41df-ae6c-eae3cc74473c.png)

**Comment by isakube** [comment]
the interpreter doesnt like it when you specify the parameters
